source/configuration.txt

```diff
@@ -435,6 +433,19 @@ The following options for writing to MongoDB are available:
 
        *Default*: 15 ms
 
+   * - ``replaceDocument``
+
+     - Replace the whole document when saving Datasets that contain an ``_id`` field.
+       If false it will only update the fields in the document that match the fields in the Dataset.
+
+       *Default*: true
+
+   * - ``maxBatchSize``
+
+     - The maximum batch size for bulk operations when saving data.
+
+       *Default*: 512
+
```

Comment:
Aside from changing version numbers, this seems like the only main content difference between v2.1 and v2.0, correct?

This may be a question for @kay-kim , but should we create a new docs branch if only for these changes?